core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -469,9 +469,12 @@ <h2>Suggestions to review</h2>
                       <div class="col-lg-12 col-md-12 col-sm-12">
                         <div style="white-space: pre-wrap;"><[m.text]></div>
                         <div ng-if="activeThread.isSuggestionThread() && $index == 0">
-                          <button class="btn btn-primary" style="margin-top: 6px; margin-bottom: 6px" ng-click="showSuggestionModal()">
+                          <button class="btn btn-primary" style="margin-top: 6px; margin-bottom: 6px" ng-click="showSuggestionModal(0)">
```

Comment:
instead of 0 or 1, can we pass true or false? That way `$scope.suggestionEditorIsShown` can be directly assigned based on the parameter. We don't need a second if-else in the function.

Reply:
ok sir